packages/server/test/react.test.tsx

```diff
@@ -673,7 +673,7 @@ describe('useMutation()', () => {
       trpc.useMutation(['deletePosts'], {
         onMutate: () => 'foo' as const,
         onSuccess: (_data, _variables, context) => {
-          expectTypeOf(context).toMatchTypeOf<'foo'>();
+          expectTypeOf(context).toMatchTypeOf<'foo' | undefined>();
```

Comment:
Revert? Is this related?

Reply:
> Tests appear to be failing on unchanged files? (https://github.com/trpc/trpc/pull/2085#issuecomment-1176059766)

[trpc/trpc@`f19ede4` (#2085)](https://github.com/trpc/trpc/pull/2085/commits/f19ede4a67721d882602d75471c12ff506a3857c)